back/app/policies/basket_policy.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class BasketPolicy < ApplicationPolicy
+  # NOTE: If we change any of these, we also need to change BasketsIdeaPolicy
```

Comment:
I think line 16 or 17 would be better, because if update no longer corresponds to create, we might get an issue